src/storm-pomdp/transformer/ApplyFiniteSchedulerToPomdp.cpp

```diff
@@ -31,7 +31,7 @@ struct RationalFunctionConstructor {
 };
 
 template<typename ValueType>
-std::shared_ptr<RawPolynomialCache> getCache(storm::models::sparse::Pomdp<ValueType> const& model) {
+std::shared_ptr<RawPolynomialCache> getCache([[maybe_unused]] storm::models::sparse::Pomdp<ValueType> const& model) {
```

Comment:
In this instance, I would have expected to just dont name the model, see https://github.com/moves-rwth/storm/issues/461

Reply:
I see your point. My idea to do it like that was that the specific instantiation with storm::RationalFunction requires the model parameter, so simply deleting it would not work. I could rework its usages?

Reply:
I think Sebastian meant to only remove the variable name, either:
```
std::shared_ptr<RawPolynomialCache> getCache(storm::models::sparse::Pomdp<ValueType> const&) {
```
or 
```
std::shared_ptr<RawPolynomialCache> getCache(storm::models::sparse::Pomdp<ValueType> const& /*model*/) {
```

Reply:
Ah, I actually didn't know that that's possible. Thanks, I will change it!

Reply:
Thanks